packages/app-transfer/src/Transfer.tsx

```diff
@@ -138,11 +163,62 @@ class Transfer extends React.PureComponent<Props, State> {
         amount,
         extrinsic,
         hasAvailable,
+        maxBalance,
         recipientId
       };
     });
   }
 
+  private setMaxBalance = () => {
+    const { api, balances_fees = ZERO_FEES } = this.props;
+    const { accountId, recipientId } = this.state;
+
+    if (!accountId || !recipientId) {
+      return;
+    }
+
+    void api.query.system.accountNonce(accountId, (accountNonce) => {
+
+      const { transferFee, transactionBaseFee, transactionByteFee, creationFee } = balances_fees;
+
+      void api.derive.balances.all(accountId, ({ availableBalance: senderBalance }) => {
```

Comment:
Not sure about the `void api.derive.balances`? Why the `void`?

Reply:
Linting error without it :(